packages/react-slider/package.json

```diff
@@ -26,33 +26,41 @@
     "start-test": "just-scripts jest-watch"
   },
   "devDependencies": {
-    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
+    "@fluentui/eslint-plugin": "^1.0.0",
     "@fluentui/react-conformance": "^0.2.1",
     "@types/enzyme": "3.10.3",
     "@types/enzyme-adapter-react-16": "1.0.3",
     "@types/react": "16.9.42",
     "@types/react-dom": "16.9.10",
     "@types/webpack-env": "1.16.0",
     "@fluentui/scripts": "^1.0.0",
-    "@fluentui/jest-serializer-merge-styles": "^8.0.0-beta.7",
+    "@fluentui/jest-serializer-merge-styles": "^8.0.0",
     "enzyme": "~3.10.0",
     "enzyme-adapter-react-16": "^1.15.0",
     "react": "16.8.6",
     "react-app-polyfill": "~1.0.1",
     "react-dom": "16.8.6"
   },
   "dependencies": {
-    "@fluentui/react": "^8.0.0-beta.62",
-    "@fluentui/react-hooks": "^8.0.0-beta.15",
-    "@fluentui/set-version": "^8.0.0-beta.2",
-    "@fluentui/style-utilities": "^8.0.0-beta.19",
-    "@fluentui/utilities": "^8.0.0-beta.15",
+    "@fluentui/react": "^8.0.0",
+    "@fluentui/react-hooks": "^8.0.0",
+    "@fluentui/set-version": "^8.0.0",
+    "@fluentui/style-utilities": "^8.0.0",
+    "@fluentui/utilities": "^8.0.0",
     "tslib": "^1.10.0"
   },
   "peerDependencies": {
     "@types/react": ">=16.8.0 <18.0.0",
     "@types/react-dom": ">=16.8.0 <18.0.0",
     "react": ">=16.8.0 <18.0.0",
     "react-dom": ">=16.8.0 <18.0.0"
+  },
+  "beachball": {
```

Comment:
The same story as for `react-cards` & `react-checkbox`, I don't know if it worth bumping for v8 release as these packages will be used in convergence soon.